scripts/generate_llms.sh

```
@@ -0,0 +1,708 @@
+if [[ -n "${OUTPUT_PATH}" ]]; then
+  OUTPUT_SINK="$STAGING_PATH"
+  : > "$OUTPUT_SINK"
+else
+  OUTPUT_SINK=""
+fi
+write_line "<!-- repo: ${REPO_SLUG} -->"
+write_line "<!-- ref: ${DEFAULT_BRANCH} -->"
+
+# Build changed files set from GitHub compare if provided
+CHANGED_SET="${SECTION_TMP_DIR}/changed_paths.txt"
+> "$CHANGED_SET"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)